tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/shadow.ts

```diff
@@ -0,0 +1,35 @@
+import { Species } from "./base";
+
+const Shadowperson: Species = {
+  description: "Victims of a long extinct space alien. Their flesh is a sickly \
+    seethrough filament, their tangled insides in clear view. Their form \
+    is a mockery of life, leaving them mostly unable to work with others under \
+    normal circumstances.",
+  features: {
+    good: [{
+      icon: "moon",
+      name: "Shadowborn",
+      description: "Their skin blooms in the darkness. All kinds of damage- \
+      no matter how extreme- will heal over time as long as there is no light.",
+    }, {
+      icon: "eye",
+      name: "Nightvision",
+      description: "Their eyes have adapted to the night. Their red eyes can \
+      see in the dark with no problems.",
```

Comment:
Very odd wording here, I'd replace with just "Their eyes, adapted to the night, can see in the dark with no problems."